README.md

```diff
@@ -62,10 +100,10 @@ Compiles and minifies for production
 npm run build
 ```
 
-Lints and fixes files
+Compiles and hot-reloads for development
 
 ```
-npm run lint
+npm run serve
```

Comment:
work-event-project|master ⇒ npm run serve
npm ERR! missing script: serve

there are no `serve` and `lint` commands in package.json